addons/html_editor/static/src/editor.js

```diff
@@ -94,8 +94,7 @@ export class Editor {
         this.editable = editable;
         this.document = editable.ownerDocument;
         if (this.config.content) {
-            const content = fixInvalidHTML(this.config.content);
-            editable.innerHTML = content;
+            editable.innerHTML = this.config.content;
```

Comment:
I guess the new location where you call the function is more global than this one so you don't need this one anymore?

Reply:
Exactly, i dont need to fix the html here because it was fixed and parse at this stage.